stock_reception_screen/models/stock_reception_screen.py

```diff
@@ -656,6 +656,53 @@ def button_save_step(self):
             return
         method = "process_{}".format(self.current_step)
         getattr(self, method)()
+        return True
+
+    def button_next_pack(self):
```

Comment:
The whole logic below could be done in a while loop checking the current step and performs the relevant operations, this to avoid to duplicate the order of the steps from the global class attribute... But I preferred to keep the thing simple for now.